src/test/java/org/kie/processmigration/persistence/GenerateDDLScriptsTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.processmigration.persistence;
+
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import javax.persistence.Persistence;
+
+import org.hibernate.engine.jdbc.internal.DDLFormatterImpl;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * Utility class for generating DDL scripts (create and drop) please ignore it.
+ */
+@Disabled
+public class GenerateDDLScriptsTest {
+
+    public static class ScriptFile {
+
+        private final String dialect;
+        private final String alias;
+        private final String prefix;
+
+        public ScriptFile(String dialect, String alias, String prefix) {
+            this.dialect = dialect;
+            this.alias = alias;
+            this.prefix = prefix;
+        }
+
+        public Path buildCreateFile(Path basePath) {
+            return basePath.resolve(alias).resolve(prefix + "-pim-schema.sql");
+        }
+
+        public Path buildDropFile(Path basePath) {
+            return basePath.resolve(alias).resolve(prefix + "-pim-drop-schema.sql");
+
+        }
+
+        public String getDialect() {
+            return this.dialect;
+        }
+
+        @Override
```

Comment:
You can remove these methods and use lombok that is already included

Reply:
yes, thanks for that